barretenberg/cpp/src/barretenberg/ultra_honk/oink_prover.cpp

```diff
@@ -168,27 +145,19 @@ template <IsUltraFlavor Flavor> void OinkProver<Flavor>::execute_sorted_list_acc
     // Commit to lookup argument polynomials and the finalized (i.e. with memory records) fourth wire polynomial
     {
         PROFILE_THIS_NAME("COMMIT::lookup_counts_tags");
-        witness_commitments.lookup_read_counts =
-            proving_key->proving_key.commitment_key->commit(proving_key->proving_key.polynomials.lookup_read_counts);
-        witness_commitments.lookup_read_tags =
-            proving_key->proving_key.commitment_key->commit(proving_key->proving_key.polynomials.lookup_read_tags);
+        commit_to_witness_polynomial(proving_key->proving_key.polynomials.lookup_read_counts,
```

Comment:
Can you please turn on the use of `commit_sparse` here? Its not being used in master either but it definitely should be used